docs/src/main/sphinx/connector/memsql.rst

```diff
@@ -7,6 +7,14 @@ creating tables in an external SingleStore database. The SingleStore connector
 is very similar to the MySQL connector with the only difference being the
 underlying driver.
 
+Requirements
+------------
+
+To connect to SingleStore, you need:
+
+* SingleStore version 7.3 or higher.
```

Comment:
3.0.0 or higher

Based on default tag in https://github.com/trinodb/trino/blob/master/plugin/trino-memsql/src/test/java/io/trino/plugin/memsql/TestingMemSqlServer.java#L33

Reply:
That tag structure is a bit misleading. The MemSQL version is 7.1.13 in that tag. The suffix is the MemSQL Cluster-in-a-box tooling version - which is unrelated to MemSQL version being used.